tests/integration/helm/upgrade/helm_upgrade_test.go

```diff
@@ -18,16 +18,14 @@
 package helmupgrade
 
 import (
-	"fmt"
-	"strings"
 	"testing"
 
 	"istio.io/istio/pkg/test/framework"
 )
 
 const (
-	previousSupportedVersion = "1.11.3"
-	nMinusTwoVersion         = "1.10.0"
+	previousSupportedVersion = "1.16.0"
```

Comment:
is there a way to get rid of this hardcoding? and automatically take the latest version and a Minus Version?

Reply:
I think there's a 1.17-dev in the makefile. I can see if there's a variable in a later or. I'm planning to clean this up more but didn't want to do it until it's up to date.